projects.go

```diff
@@ -83,8 +83,9 @@ type Project struct {
 		GroupName        string `json:"group_name"`
 		GroupAccessLevel int    `json:"group_access_level"`
 	} `json:"shared_with_groups"`
-	Statistics *ProjectStatistics `json:"statistics"`
-	Links      *Links             `json:"_links,omitempty"`
+	Statistics   *ProjectStatistics `json:"statistics"`
+	Links        *Links             `json:"_links,omitempty"`
+	CiConfigPath *string            `json:"ci_config_path"`
```

Comment:
Consider capitalizing the `I` as in `CIConfigPath`. Go likes capitalized acronyms 
Ref: https://github.com/golang/go/wiki/CodeReviewComments#initialisms

Reply:
I copied this from the already-existing https://github.com/xanzy/go-gitlab/blob/062bdff086ddaf6a8de7335bcd57bf7d468c4967/projects.go#L353 — shouldn’t we keep the two consistent?

Reply:
Ah... Guess that one slipped through 😞  But I rather fix the other one in that case...